fastai/vision/augment.py

```diff
@@ -489,7 +541,12 @@ def encodes(self, x:TensorImage):
         return TensorImage(x).affine_coord(sz=self.size, mode=self.mode)
 
 # Cell
-def mask_tensor(x, p=0.5, neutral=0., batch=False):
+def mask_tensor(
+    x:Tensor, # Input `Tensor`
+    p=0.5, # Propability of not masking an element of `x`
+    neutral=0., # The mask value
+    batch=False # Same behavior for the whole batch if `True`
+):
```

Comment:
More changes to be made to all the GPU helpers as a suggestion here: Modify doc strings for conciseness & specificity.

```suggestion
def mask_tensor(
    x:Tensor, # Input `Tensor`
    p=0.5, # Probability of not applying mask
    neutral=0., # Mask value
    batch=False # Apply identical mask to entire batch
):
```